### ISSUE TITLE
Answer with Yes and not OK to an interrogative sentence

### ISSUE THREAD

**Issue by summoner001** [Anuken/Mindustry#10093]
### Platforms

Linux

### Build

25168

### Issue

There are two kinds of interrogation sentences in the Mindustry bundle file. One kind is a simple yes or no answer to a question that needs to be decided, we don't use for the answer with OK (I have no objection)

The other type of interrogation sentence for which the answer is OK. There are only two of this type in Mindustry as far as I have looked.
909-settings.clearall.confirm=...
1219-uiscale.reset=...
The Hungarian language only answers these two questions or statements with OK (I have no objection)

For the other questions in Mindustry, Hungarian can answer No (or special case: Cancel) or Yes.

The hungarian here would need a Yes (Igen) button for the answer to the decided question and an OK (Rendben) button when there is no objection.

Example:
96-map.delete = Are you sure you want to delete the map "[accent]{0}[]"?
Answer: Cancel / Yes -> Igen

909-settings.clearall.confirm=[scarlet]WARNING![]\nThis will clear all data, including saves, maps, unlocks and keybinds.\nOnce you press 'ok' the game will wipe all data and automatically exit.
1219-uiscale.reset=...
Answer: Cancel / OK (I have no objection) -> Rendben

### Steps to reproduce

No steps, this is a string shortcoming

### Submission

- [X] I have updated to the latest release (https://github.com/Anuken/Mindustry/releases) to make sure my issue has not been fixed.
- [X] I have searched the closed and open issues to make sure that this problem has not already been reported.
- [X] I am not using Foo's Client, and have made sure the bug is not caused by mods I have installed.

**Comment by Matita008** [comment]
This repo is not for suggestions

**Comment by summoner001** [comment]
This is not a suggestion.

This is a Hungarian language rule.

Therefore this is a localization problem in hungarian bundle. 🙂

**Comment by Matita008** [comment]
You know you can pr it instead of opening a bug report?

**Comment by summoner001** [comment]
Yes I know.
But in the Hungarian language we cannot answer a question that needs to be decided by saying OK only with yes or no. We understand but it is not a habit and a mistake.

**Comment by summoner001** [comment]
Example:
Question: Do you want coffee?
Answer: Yes/No (Yes I do/No I don't)
We don't use the word "ok".

**Comment by summoner001** [comment]
The yes text is missing from the bundle but the other two sentences have the word ok.
I'm not a programmer, I can't do it.

**Comment by Anuken** [comment]
If usage of "okay" is the issue, can't the text be rephrased in a more passive way for your bundle?

e.g. "Are you sure you want to delete the map?" -> "Map {0} will be deleted." [OK]

**Comment by summoner001** [comment]
I'll see what I can do and think about how I can.

**Comment by summoner001** [comment]
I rewrote the question sentences in different words.
Here it is my PR: https://github.com/Anuken/Mindustry/pull/10099

Sorry I NOOB for github. I know I should link the issu to PR now. Successfully merging this pull request may close these issues.
But I can't do it now for some reason and it's been a long time since I've done it once and I don't remember how.

**Comment by summoner001** [comment]
I tried to rewrite the question sentences, but eventually I realised that they should not be rewritten.

Two reasons:

1) if someone starts translating after me, they won't understand what happened to the question sentences and will see it as a mistake and will try to correct it.
2) the sentences became longer after the rewrite, and nobody likes to read a lot while playing.

Another solution:

There is one solution to fix the question sentences and that is to introduce a Yes answer in the answer options instead of OK. However, this only applies to sentences ending in ?. The other sentences should remain with the OK answer option.

Yes I know, there is no time for this and it is not such a fatal mistake. Unfortunately I can't do it because I'm a programming noob. It will be done when it's done, it's not that important.
